templates/README.md

```diff
@@ -10,29 +10,15 @@ Questions about Workers? Please join the [Cloudflare Developers Discord](https:/
 
 There are a few ways to quickly jumpstart your next project using one of the templates found within this repository:
 
-<!-- todo(eidam): is workers.new/<name> still hardcoded or ready for all? -->
-<!-- 2. In-browser development, via `workers.new` -->
-
 1. **Local development, via CLI quickstart utility**
 
-   You may use `yarn`, `pnpm`, or `npm` to invoke the [`create-cloudflare`](https://www.npmjs.com/package/create-cloudflare) package.
-
-   > **Note:** All recent versions of `npm`, `yarn`, and `pnpm` support this feature!
+   We recommend using the `npx create cloudflare@1 [folder-name] [template-name]` command to create new projects with templates.
```

Comment:
Isn't it `@2` now?

Reply:
@rozenmd `@2` will not currently work as expected to create templates this way. As a workaround, users wanting to copy/paste as outlined here should use v1 until c3 supports this.